scripts/chgres_cube.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python
+"""
+The run script for chgres_cube
+"""
+
+import datetime as dt
+import logging
+import os
+import re
+import sys
+from argparse import ArgumentParser
+from copy import deepcopy
+from pathlib import Path
+
+from uwtools.api.chgres_cube import ChgresCube
+from uwtools.api.config import get_yaml_config
+from uwtools.api.fs import link as uwlink
+from uwtools.api.logging import use_uwtools_logger
+
+
+def _parse_var_defns(file):
```

Comment:
Sorry, I forgot to circle back to this point. I thought we'd discussed changing this file's format to YAML so that we wouldn't have to support a non-standard format.

Reply:
No worries, I hadn't added that, I'll make sure that gets updated